### PR TITLE
Feature branch to Development branch

### PR DESCRIPTION
**Pull request by AdedayoOpeyemi**
Re-former

The project is to build forms using the helper methods that Rails provides.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @AdedayoOpeyemi and @aldemeery 
### You have done well so far :clap: However, you have an issue to fix
- [x] You are required to use `form_for` when using forms in rails - best practice
- [x] Comment out the old form

By Raymond
slack @iamraymond

File: app/views/users/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  <div class="field">
+    <%= form.label :password %>
+    <%= form.password_field :password %>
+  </div>
+
+  <div class="actions">
+    <%= form.submit %>
+  </div>
+<% end %>
```
Comment (rOluochKe) [inline app/views/users/_form.html.erb:32]: - [ ] Use ruby `form_for` and comment out the usage of `form_tag` and `form_with`

**Comment by aldemeery** [comment]
Hello @rOluochKe 

I think you have mistaken here...
`form_for` is not the best practice, if this is mentioned somewhere in a guide you follow when reviewing PRs, then I suggest
you ask for that guide to be updated.

From the [official Rails documentation](https://guides.rubyonrails.org/form_helpers.html#using-form-for-and-form-tag), you can see that `form_for` and `form_tag` are both now soft-deprectated, and both have been replaced with `form_with`

image

**Review by mosesogwo — APPROVED** [review]
**Hello! :clap: Great job! Your project is approved**
Please note that it's best to stick with the project specifications (provided on Pathwright and Odin) in order to reduce conflicts.

_**Good luck in your next project:stars:**_

### Status: Approved :heavy_check_mark:

**Pull request merged**